yarn.lock

```diff
@@ -15634,7 +15634,7 @@ rollup@^2.43.1, rollup@^2.58.0:
   optionalDependencies:
     fsevents "~2.3.2"
 
-rollup@endojs/endo#rollup-2.7.1-patch-1:
+"rollup@github:endojs/endo#rollup-2.7.1-patch-1":
```

Comment:
seems like some different Yarn versions in play. is yours `1.22.19`?

Reply:
Indeed, I’m on `1.22.19` and Node.js `14.19.0`. 🧐